openedx/core/djangoapps/xmodule_django/models.py

```diff
@@ -81,8 +81,6 @@ class OpaqueKeyField(models.CharField):
     """
     description = "An OpaqueKey object, saved to the DB in the form of a string."
 
-    __metaclass__ = models.SubfieldBase
-
```

Comment:
@ormsbee This `SubfieldBase` removal is being done for the Django upgrade - and is a relatively significant change. (See the commit message.) Could you please review this commit? Thanks!